clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -534,12 +521,12 @@ def create_file(image, modality, bids_dir, overwrite):
 
     # creation of the path
     if modality == "t1":
-        output_path = join(bids_dir, f"sub-AIBL{subject}", f"ses-{session}", "anat")
-        output_filename = f"sub-AIBL{subject}_ses-{session}_T1w"
+        output_path = join(bids_dir, f"sub-AIBL{subject}", f"{session}", "anat")
+        output_filename = f"sub-AIBL{subject}_{session}_T1w"
```

Comment:
It would be better to factor the computation of `participant_id` and `session_id` and use these names to avoid ambiguity. When I see `subject` or `session` I can't know whether I am dealing with the label or the ID unless I look at the rest of the implementation.

```suggestion
        output_path = join(bids_dir, f"{participant_id}", f"{session_id}", "anat")
        output_filename = f"{participant_id}_{session_id}_T1w"
```